cmd/server/app/server.go

```diff
@@ -249,8 +249,11 @@ func (p *Proxy) runMTLSMasterServer(ctx context.Context, o *options.ProxyRunOpti
 	addr := fmt.Sprintf(":%d", o.ServerPort)
 
 	if o.Mode == "grpc" {
-		serverOption := grpc.Creds(credentials.NewTLS(tlsConfig))
-		grpcServer := grpc.NewServer(serverOption)
+		masterServerOptions := []grpc.ServerOption{
+			grpc.Creds(credentials.NewTLS(tlsConfig)),
+			grpc.KeepaliveParams(keepalive.ServerParameters{Time: o.KeepaliveTime}),
```

Comment:
I believe this should be using the timeout set in MasterKeepaliveTime.

Reply:
correct, updated